src/ui/map.js

```diff
@@ -187,7 +187,7 @@ const defaultOptions = {
  * @param {number} [options.maxZoom=22] The maximum zoom level of the map (0-24).
  * @param {number} [options.minPitch=0] The minimum pitch of the map (0-85).
  * @param {number} [options.maxPitch=85] The maximum pitch of the map (0-85).
- * @param {Object|string} [options.style] The map's Mapbox style. This must be an a JSON object conforming to
+ * @param {Object|string} [options.style=null] The map's Mapbox style. This must be an a JSON object conforming to
```

Comment:
Though not directly related to the topic of this PR, the addition of these default values would be a very nice addition to the docs!